DOCUMENTATION.md

```diff
@@ -564,6 +565,31 @@ const client = await Client.newHazelcastClient(cfg);
 
 In the following chapters you will learn the description of all options supported by Hazelcast Node.js client.
 
+# 3.1 Client Properties
+
+The following is the list of all client properties in alphabetical order.
+
+> **NOTE: When you want to reconfigure a client property, you need to restart the client.**
+
+| Property Name                                          | Default Value                       | Type            | Description                                                                                                                                                                                                                                                                                                                                                                                                         |
+|--------------------------------------------------------|-------------------------------------|-----------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| hazelcast.client.autopipelining.enabled                | true                                | boolean         | Turns automated pipelining feature on/off. If your application does only write operations, like `IMap.set()`, you can try disabling automated pipelining to get a slightly better throughput.                                                                                                                                                                                                                       |
+| hazelcast.client.autopipelining.threshold.bytes        | 65536 (64 KB)                       | number          | This is the coalescing threshold for the internal queue used by automated pipelining. Once the total size of operation payloads taken from the queue reaches this value during batch preparation, these operations are written to the socket. Notice that automated pipelining will still send operations if their total size is smaller than the threshold and there are no more operations in the internal queue. |
+| hazelcast.client.cloud.url                             | https://coordinator.hazelcast.cloud | string          | Hazelcast Cloud URL.                                                                                                                                                                                                                                                                                                                                                                                                | <!-- markdown-link-check-disable-line -->
```

Comment:
This shouldn't be public. It is only suitable for the cloud team to use, to test the client in different environments. For users, they don't need to bother with this and only use the production environment, which is the default value.